Helios/Interfaces/Profile/ProfileInterface.cs

```diff
@@ -243,21 +372,54 @@ private void CheckResetMonitors(IList<StatusReportItem> newReport)
                         Recommendation = "Perform Reset Monitors from the Profile menu",
                         Severity = StatusReportItem.SeverityCode.Error,
                         Link = StatusReportItem.ProfileEditor
+                        // TODO handle reporting tablet orientated incorrectly for the profile - ie profile width = screen height and profile height = screen width - the action would be to rotate the device rather than perform a monitor reset
                     });
                 }
             }
         }
 
         /// <summary>
-        /// check configuration related to launching or stopping executables
+        /// check configuration related to launching executables
         /// </summary>
         /// <param name="newReport"></param>
         private void CheckExecutablePaths(IList<StatusReportItem> newReport)
         {
-            // TODO if any launch or kill actions are configured, check paths and access
-            // TODO if incorrect, report warning or error (to stop launch) in StatusReportItem
-            // TODO if correct, report detail also in StatusReportItem
-            _ = newReport;
+            foreach(HeliosBinding _binding in OutputBindings)
+            {
+                if(_binding.Action.ActionID == "launch application")
```

Comment:
I'd already changed this to 7.1, so moved into Platforms.properties and PlatformsForPlugin.properties

Reply:
> everyone reading it will think it refers to a field. Just use "lowerCamelCase".
Not everyone... programming troglodytes like myself wouldn't infer much from this.  Incorrect Underscores obliterated.